dev/tools/test-migration.py

```
@@ -376,10 +392,67 @@ def create_v1alpha1_objects():
+            try:
+                claims_data = json.loads(res.stdout).get("items", [])
+                bound_count = 0
+                for c in claims_data:
+                    if c["metadata"]["name"].startswith("upgrade-claim-warm") and c.get("status", {}).get("sandbox", {}).get("name"):
+                        bound_count += 1
+                if bound_count >= 11:
+                    print(f"All {bound_count} warm claims successfully bound!")
+                    break
+                if (i + 1) % 5 == 0:
+                    print(f"Waiting for warm claims to bind ({bound_count}/11 bound, attempt {i+1}/60)...")
+            except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)